js/apps/admin-ui/public/locales/no/authentication.json

```diff
@@ -18,9 +18,9 @@
     "none": "Ingen"
   },
   "flow": {
-    "browserFlow": "Nettleserflyt",
-    "registrationFlow": "Registreringsflyt",
-    "directGrantFlow": "Direct Grant Flyt"
+    "browser": "Nettleserflyt",
+    "registration": "Registreringsflyt",
+    "direct grant": "Direct Grant Flyt"
```

Comment:
we should also add these but translated  "reset credentials": "Reset credentials flow", "clients": "Client authentication flow", "docker auth": "Docker authentication flow"